contrib/babelfishpg_tsql/src/pltsql_ruleutils.c

```diff
@@ -800,6 +805,9 @@ tsql_print_function_arguments(StringInfo buf, HeapTuple proctup,
 				modename = "OUT ";
 				isinput = false;
 				break;
+			case PROARGMODE_TABLE:
+				*has_tvp = true;
+				return;
```

Comment:
Make sure to we release `bbffunctuple` (at line 882) before returning. I would recommend to `break` here and then come out of for loop with another `break` if `has_tvp` is set.